packages/autocomplete-plugin-redirect-url/src/createRedirectUrlPlugin.ts

```diff
@@ -101,6 +101,13 @@ export function createRedirectUrlPlugin<TItem extends BaseItem>(
           },
         });
       });
+
+      onSelect(({ state }) => {
+        const redirects = getRedirectData({ state });
+        if (redirects.length > 0) {
+          setIsOpen(true);
+        }
+      });
```

Comment:
I think that would be correct behaviour. The redirect plugin shouldn't prevent items from showing up or having a certain effect, _unless_ the query matches the redirect. This item doesn't match the redirect, but the query it would put does have a redirect, so if it's pre-filled, the panel is reopened, to make it clear another action is possible. If the user navigates that would be correct.

Maybe a follow-up backend could be preventing objects that match the redirect query from even showing up, but that's not yet possible I think

Reply:
Makes sense, thank you!

Reply:
Very good point. I could see this being confusing for users; however, there are limitations either way we take this. At least in this case the user technically has full control over the behavior and we're not trying to override their template behavior. 

I just chatted with Benoit about this. He thinks we should move forward with what we have and go from there. Perhaps user feedback will give us better insight with how to tackle this problem in the next iteration.